main.rb

```diff
@@ -0,0 +1,8 @@
+require_relative 'app'
+
+def main
+  app = App.new
+  app.run
+end
+
+main
```

Comment:
- [ ] Good job so far, your app work as expected :100: But I have to mention something :cry: From the project requirement, your `main.rb` define the entry point, a method called main that is invoked at the end of your file and this method should present to the user a list of options to perform, Let's choose an option, ask for parameters for the option, Have a way to quit the app. I noticed that you did it in your `app.rb` but note that as a developer it is very important to always follow the requirements because they establish a foundation for product vision, scope, cost, and schedule and they ultimately must target finished product quality and performance. Please fix it :+1: 

![Screenshot from 2022-11-03 20-23-34](https://user-images.githubusercontent.com/6615394/199826716-e9e4dc61-9b0d-482d-8c04-5dd72a870813.png)